src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -183,14 +186,30 @@ public void testGetAppealByPenaltyReference_returnsAppeal() {
         assertEquals(TestData.Appeal.Reason.Attachment.size, appeal.getReason().getOther().getAttachments().get(0).getSize());
     }
 
+    @Test
+    public void testGetMultipleAppealByPenaltyReference_returnsAppeal() {
+
+        when(appealRepository.findByPenaltyReference(any(String.class))).thenReturn(List.of(
+            createAppealEntity(TestData.Appeal.PenaltyIdentifier.penaltyReference),
+            createAppealEntity(TestData.Appeal.PenaltyIdentifier.penaltyReference)
+        ));
+
+        when(appealMapper.map(any(AppealEntity.class))).thenReturn(createAppeal());
+
+        List<Appeal> appealList = appealService.getAppealByPenaltyReference(TestData.Appeal.PenaltyIdentifier.penaltyReference);
+
+        assertEquals(appealList.size(), 2);
```

Comment:
Please swap these two arguments, as they're not in the correct order, should be: `assertEquals(expected, actual)`